### PR TITLE
add crossSelling option to widget

### PR DESCRIPTION
**Pull request by stentrop**
### All changes meet the following requirements
- [ ] Changelog entry was added
- [ ] Changes have been tested
- [ ] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

**Comment by stentrop** [comment]
@fmutschler

**Review by Dominik2809 — APPROVED** [review]

File: resources/lang/en/Widget.properties [hunk 1]
```diff
@@ -169,10 +171,17 @@ itemListItemsPerPage4 = "4"
 itemListPlaceholderItemName = "Item name"
 itemListPlaceholderItemPrice = "Item price"
 itemListPlaceholderListHeader = "Item list"
+itemListPlaceholderTitle = ""
+itemListPlaceholderDescription = ""
```
Comment (Dominik2809) [inline resources/lang/en/Widget.properties]: Warum leer?

**Review by Dominik2809 — CHANGES_REQUESTED** [review]

File: resources/lang/en/Widget.properties (hunk 1 above)
Comment (stentrop) [reply resources/lang/en/Widget.properties]: @fmutschler hat dafür eine Karte auf unserem Board

**Review by fmutschler — APPROVED** [review]
Habs sprachlich korrigiert

**Review by Dominik2809 — APPROVED** [review]

**Review by Dominik2809 — APPROVED** [review]

**Pull request merged**